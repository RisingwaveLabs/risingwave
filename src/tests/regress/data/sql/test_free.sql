--
-- TEST_FREE --- drops the tables created during setup that other tests may use
--

--@ DROP TABLE CHAR_TBL

DROP TABLE FLOAT8_TBL;

DROP TABLE INT2_TBL;
DROP TABLE INT4_TBL;
DROP TABLE INT8_TBL;

--@ DROP TABLE POINT_TBL;

DROP TABLE TEXT_TBL;

DROP TABLE VARCHAR_TBL;

DROP TABLE onek;
DROP TABLE onek2;

DROP TABLE tenk1;
DROP TABLE tenk2;

--@ DROP TABLE person;
--@ DROP TABLE emp;
--@ DROP TABLE student;
--@ DROP TABLE stud_emp;
--@
--@ DROP TABLE road;
--@ DROP TABLE ihighway;
--@ DROP TABLE shighway;
