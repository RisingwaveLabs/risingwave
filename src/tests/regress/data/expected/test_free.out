--
-- TEST_FREE --- drops the tables created during setup that other tests may use
--
